### PR TITLE
Update compliance.json

### PR DESCRIPTION
**Pull request by kiley-mitti**
this just removes a compliance rule that was copied and pasted into status symbol by mistake.

### CODE REVIEW

File: src/data/compliance.json [hunk 1]
```diff
@@ -888,12 +888,6 @@
                             "status": "current",
                             "tier": 1
                         },
```
Comment (kiley-mitti) [inline src/data/compliance.json:890]:
```suggestion
                        }
```

Comment (jonathantneal) [reply src/data/compliance.json]: 👏

Comment (kiley-mitti) [reply src/data/compliance.json]: I am so. bad at on the fly json writing

**Comment by cskerbo** [comment]
@kiley-mitti Is this PR still valid or should we close it?

**Comment by kiley-mitti** [comment]
> @kiley-mitti Is this PR still valid or should we close it?

@cskerbo Still relevant. The compliance says 'switch labels' but it is under status symbol.

**Review by micahjones13 — APPROVED** [review]

**Pull request merged**